packages/polaris-schema/src/utils/merge-types.ts

```diff
@@ -22,3 +29,17 @@ export const getMergedPolarisTypes = (types: ITypeDefinitions): string =>
             all: true,
         },
     );
+};
+
+function getDirectivesAndScalarsTypeDefs(polarisSchemaConfig: PolarisSchemaConfig) {
+    const directivesAndScalarsTypeDefs: any[] = [];
+    if (polarisSchemaConfig.addPolarisDirectives) {
+        directivesAndScalarsTypeDefs.push(directivesTypeDefs);
```

Comment:
in my opinion you should name `directivesTypeDefs` to permissionTypeDefs and add it only if the permissionconfig was provided